src/lib/modules/Loan/backoffice/OverdueLoanSendMailModal/state/reducer.js

```diff
@@ -30,3 +31,37 @@ export default (state = initialState, action) => {
       return state;
   }
 };
+
+export const initialFetchState = {
+  isLoading: false,
+  hasError: false,
+  reminders: {},
+  error: {},
+};
+
+export const fetchOverdueLoanRemindersReducer = (
```

Comment:
why in the end we went for the redux state? Is this component sharing the state with any others?